### DISCUSSION TITLE
How much is Scrypted using the file system?

### DISCUSSION THREAD

**Discussion by tillkruss** [koush/scrypted#1249] [General]
Hey 👋 

Absolutely love Scrypted and it's UniFi to HomeKit Secure Video integration.

The container clearly uses RAM and some CPU, but I was wondering how much disk I/O Scrypted is doing?

It appears as if the "video buffering" and HSV motion detection only happens in RAM?

**Comment by koush** [comment]
not using file system at all really

**Reply by tillkruss** [reply]
Fantastic, I was hoping for that.
